src/main/java/com/google/flourbot/api/DriveCloudSheet.java

```diff
@@ -12,49 +12,66 @@
 import java.util.List;
 
 public class DriveCloudSheet implements CloudSheet {
-  Sheets sheetsService;
-  String spreadsheetId;
+  private String valueInputOption = "USER_ENTERED";
```

Comment:
example of a proper constant in Java:
private static final String VALUE_INPUT_OPTION = "USER_ENTERED";

Reply:
Whoops that's a silly mistake - thanks for pointing it out!